.github/CODEOWNERS

```diff
@@ -52,7 +52,11 @@ privacy-snapshot.json                @MetaMask/extension-privacy-reviewers
 .devcontainer/                       @MetaMask/library-admins @HowardBraham @plasmacorral
 
 # Confirmations team to own code for confirmations on UI.
-ui/pages/confirmations               @MetaMask/confirmations
+ui/pages/confirmations                  @MetaMask/confirmations
```

Comment:
Not blocking, but curious if we should alter the rules to have less strict ownership over `ui/pages/confirmations/confirmation/templates`? It's not clear to me if you want to be forced to approve changes to templates or that is a side-effect of the current rules 😄

Reply:
Agree that it makes sense for teams to own templates with a clear related domain such as the snaps templates.

Though perhaps simplest to leave this entry as a default so teams can add entries below to override ownership for specific template directories and snapshots?